Lib/ufo2ft/util.py

```diff
@@ -243,13 +243,18 @@ def makeUnicodeToGlyphNameMapping(font, glyphOrder=None):
     return mapping
 
 
-def compileGSUB(featureFile, glyphOrder):
+def compileGSUB(featureFile, glyphOrder, fvar=None):
     """Compile and return a GSUB table from `featureFile` (feaLib
     FeatureFile), using the given `glyphOrder` (list of glyph names).
     """
     font = ttLib.TTFont()
     font.setGlyphOrder(glyphOrder)
-    addOpenTypeFeatures(font, featureFile, tables={"GSUB"})
+    if fvar:
+        font["fvar"] = fvar
+        tables = {"GSUB", "GDEF"}
```

Comment:
can you add a comment why you're also building GDEF here when fvar is not None?

Reply:
Not needed.